lib/vtls/schannel.c

```diff
@@ -1576,6 +1610,159 @@ CURLcode Curl_schannel_random(unsigned char *entropy, size_t length)
   return CURLE_OK;
 }
 
+static CURLcode pkp_pin_peer_pubkey(struct connectdata *conn, int sockindex,
```

Comment:
I would recommend the same as in https://github.com/curl/curl/pull/1325#discussion_r112832287 and would like to see such a function moved to a separate source file, e.g. schannel_cert.c.